src/frontend/screens/Settings/ProjectConfig/ManagePeople.tsx

```diff
@@ -0,0 +1,157 @@
+import * as React from "react";
+import { StyleSheet, View } from "react-native";
+import { useNavigation } from "react-navigation-hooks";
+import { getUniqueId } from "react-native-device-info";
+import MaterialIcon from "react-native-vector-icons/MaterialIcons";
+import { FormattedMessage, defineMessages } from "react-intl";
+
+import { MAPEO_BLUE } from "../../../lib/styles";
+import { showWipAlert } from "../../../lib/utils";
+import Button from "../../../sharedComponents/Button";
+import Text from "../../../sharedComponents/Text";
+
+import { MemberRow } from "./MemberRow";
+
+const m = defineMessages({
+  managePeople: {
+    id: "screens.Settings.ManagePeople.managePeople",
+    defaultMessage: "Manage People",
+  },
+  add: {
+    id: "screens.Settings.ManagePeople.add",
+    defaultMessage: "Add",
+  },
+  leaveProject: {
+    id: "screens.Settings.ManagePeople.leaveProject",
+    defaultMessage: "Leave Project",
+  },
+  remove: {
+    id: "screens.Settings.ManagePeople.remove",
+    defaultMessage: "Remove",
+  },
+  yourDevice: {
+    id: "screens.Settings.ManagePeople.yourDevice",
+    defaultMessage: "(You) {deviceName}",
+  },
```

Comment:
Lets add a description for this one as it is a little ambiguous if we dont have the context